rascal-lsp/src/main/rascal/lsp/lang/rascal/lsp/refactor/Rename.rsc

```diff
@@ -69,10 +69,14 @@ import util::LanguageServer;
 import util::Maybe;
 import util::Reflective;
 
+private bool isQualifiedUse(loc use, Define def) = size(def.id) != use.length;
```

Comment:
Is this logic sound? No way the lengths can differ without use being qualified?

Reply:
The escape is not included in the location:
```rascal
void main() {
    int foo = 1;
    bar = \foo; // hold control and hover -> the \ is not part of the hyperlink
}
```

Added some additional tests for this nonetheless.